File: SEP3_T2/grpc-springboot/src/main/java/sep3/dto/CategoryDTO.java
package sep3.dto;

public class CategoryDTO {
    private String name;
    private int id;

    private int addedBy;

    public CategoryDTO(String Name, int Id, int AddedBy) {
        this.name = Name;
        this.id = Id;
        this.addedBy = AddedBy;
    }

    public String getName() {
        return name;
    }

    public int getId() {
        return id;
    }

    public int getAddedBy() {
        return addedBy;
    }
}

File: SEP3_T1/Shared/Domain/DTOs/Category/CategoryDTO.cs
namespace DTOs.User;

public class CategoryDTO
{
    public int Id { get; set; }
    public string Name { get; set; }
    public int addedBy { get; set; }
    
    public CategoryDTO(int id, string name, int addedBy)
    {
        this.Id = id;
        this.Name = name;
        this.addedBy = addedBy;
    }
}